schema-dex-amm.graphql

```diff
@@ -1,20 +1,27 @@
 # Subgraph Schema: DEX AMM
-# Version: 1.2.1
+# Version: 1.2.2
 # See https://github.com/messari/subgraphs/blob/master/docs/Schema.md for details
 
 enum Network {
   ARBITRUM_ONE
+  ARWEAVE_MAINNET
   AURORA
   AVALANCHE
+  BOBA
   BSC # aka BNB Chain
   CELO
+  COSMOS
+  CRONOS
```

Comment:
Should `Clover` be added here too? https://parachains.info/details/clover_finance

Reply:
hmm, I can't even find Clover in Defillama. Are they relevant?